packages/types/src/type/ExtrinsicEra.ts

```diff
@@ -2,33 +2,224 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
+import { assert, hexToU8a, isHex, isU8a, isObject, u8aToBn } from '@polkadot/util';
 import { AnyU8a } from '../types';
+import Enum from '../codec/Enum';
+import Tuple from '../codec/Tuple';
+import U8 from '../primitive/U8';
+import U8a from '../codec/U8a';
 
-import { u8aToU8a } from '@polkadot/util';
+export type MortalEraValue = [U8, U8];
```

Comment:
Does this need to be exported? Cannot see it used externally from a quick search.

Reply:
no, shouldn't be exported..changed